keith-vscode/src/simulation/simulation-table-data-provider.ts

```diff
@@ -472,7 +467,7 @@ export class SimulationTableDataProvider implements vscode.WebviewViewProvider {
         }
     }
 
-    async handleAddCoSimulation(action: PerformActionAction): Promise<void> {
+    async handleAddCoSimulation(): Promise<void> {
```

Comment:
What is in the action that was once here?

Reply:
Just an unused parameter that is not needed, probably got added here to look like the other handle methods, that do use the action, whereas this one does not.